lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -35,20 +35,20 @@ def initialize(name, instantiate=true)
     def self.globals
       hash = {}
       is_fabricpath_feature = config_get('fabricpath', 'feature')
-      return hash if (:enabled != is_fabricpath_feature.first.to_sym)
+      return hash if (:enabled != is_fabricpath_feature.to_sym)
```

Comment:
Just a thought.  Could you rewrite the feature yaml entry to add `default_value` and `kind` features and check for enabled explicitly?

```
feature:
  default_value: false
  kind: boolean:
  config_get: "show feature-set"
  config_get_token: '/^fabricpath[\s]+[\d]+[\s]+(enabled)/'
  config_set: "%s feature-set fabricpath"
```
Then you could collapse the feature enabled check into a single line like this...

`return hash unless config_get('fabricpath', 'feature')`